examples/tcp-retransmit.bpf.c

```diff
@@ -0,0 +1,88 @@
+#include <vmlinux.h>
+#include <bpf/bpf_tracing.h>
+#include "maps.bpf.h"
+#include <bpf/bpf_core_read.h>
+
+#define MAX_ENTRIES 8192
+#define RETRANSMIT  1
+#define TLP         2
+
+#define AF_INET		2
+#define AF_INET6	10
+
+struct ipv4_key_t {
+	u32 saddr;
+	u32 daddr;
+	__u16 sport;
+	__be16 dport;
+    int type;
+};
+
+struct ipv6_key_t {
+	u8 saddr[16];
+	u8 daddr[16];
+	__be16 sport;
+	__u16 dport;
+    int type;
+};
+
+struct {
+	__uint(type, BPF_MAP_TYPE_HASH);
```

Comment:
I would recommend `BPF_MAP_TYPE_LRU_HASH` to allow eviction of old stuff instead of dropping of new stuff.

Reply:
Changed.